CHANGES.txt

```diff
@@ -4,6 +4,7 @@ Changes for Crate Admin Interface
 
 Unreleased
 ==========
+ - Added translation for controllers.
```

Comment:
hm. for an end-user this is not really meaningful.
what about:
- added translation for Tutorial plugin. this enables support for i18n for any admin-ui plugin.